### ISSUE TITLE
Importing six in setup.py (commit 3786502) fails in clean Python2 environments

### ISSUE THREAD

**Issue by denebolar** [miguelgrinberg/python-engineio#87]
In clean environments, e.g. a freshly created virtualenv, six is not available as a dependency even though it is included in install_requirements (no pre-depend, ready only after setup finishes).

Environment tested: Python 2.7.15

**Comment by miguelgrinberg** [comment]
Duplicate. Closing in favor of https://github.com/miguelgrinberg/python-socketio/issues/235.
